app/controllers/concerns/verify_setup_completed.rb

```diff
@@ -0,0 +1,41 @@
+module VerifySetupCompleted
+  extend ActiveSupport::Concern
+
+  MANDATORY_CONFIGS = %i[
+    community_description
+    community_member_description
+    community_action
+    tag_url
+
+    main_social_image
+    favicon_url
+    logo_png
```

Comment:
We can technically gate `favicon_url`, `logo_png` or even just use a [placeholder](https://placeholder.com/) image but I feel like a logo and favicon are mandatory items that should be dealt with on setup.

Reply:
I agree on `logo_png`, not so sure about `favicon_url`, as 404s for those are common.

Reply:
If logos, favicons and app icons are configurable through admin panel, we should ask admin to upload at least two assets:

1. Logo used everywhere in UI
2. Icon being a square image that would be also used for any other icons assets (favicon, app icon, etc.) unless admin decides to overwrite that with custom ones if that makes sense.